low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -180,7 +174,7 @@ class SSPLDevDeploy:
                print(f"ERROR: Unable to download {pkg_name}.")
             cmd = "yum install -y %s%s" % (build_url, pkg[0])
             returncode = self._execute_cmd(cmd)
-            if not (returncode==0 or returncode==256):
+            if returncode != 0:
```

Comment:
Is it fine to remove returncode==256 here?
I am seeing that you have added this earlier.

Reply:
Yes, if pkg is already installed, we had two different cases.
Now we uninstall cort-py-utils from cleanup(). So we expect returncode 0 for this fresh installation.